shell/lint.sh

```diff
@@ -31,7 +31,7 @@ then
   exit 1
 fi
 [ $# -eq 0 ] && echo "no issues with flake8"
-black --line-length 80 --check $files
```

Comment:
Just to verify -- if using `lint.sh` outside of devcontainer, this setting will still be taken from `pyproject.toml` -- correct?

Reply:
Yes, that's correct.